python/cuopt_self_hosted/cuopt_sh_client/cuopt_self_host_client.py

```diff
@@ -306,13 +317,30 @@ def __init__(
         polling_timeout=600,
         timeout_exception=True,
         result_type=mime_type.MSGPACK,
+        http_general_timeout=30,
```

Comment:
I think we should enforce the type that we will accept in the parameter value, but we can tackle it in a follow-up PR.